CoreML in ARKit/ViewController.swift

```diff
@@ -82,10 +81,7 @@ class ViewController: UIViewController, ARSCNViewDelegate {
         sceneView.session.pause()
     }
     
-    override func didReceiveMemoryWarning() {
-        super.didReceiveMemoryWarning()
-        // Release any cached data, images, etc that aren't in use.
-    }
+   
```

Comment:
Cleared as most of those problems were back in the iPhone 3GS days when memory was more limited.